spec.bs

```diff
@@ -16,6 +16,10 @@ Markup Shorthands: css no, markdown yes
 Ignored Terms: h1, h2, h3, h4, h5, h6, xmp
 </pre>
 
+<pre class="anchors">
+urlPrefix: https://fetch.spec.whatwg.org/; spec: Fetch
+    text: http-network-or-cache fetch; url: #concept-http-network-or-cache-fetch; type: dfn
```

Comment:
(i guess free PR to fetch for someone who wants to export this later)

Reply:
They might not like monkeypatching and block this on principle 🙃